python/README.md

```diff
@@ -2,4 +2,13 @@ This folder contains the Python bindings and code required for embedding python
 
 For the Python bindings, see the [Python bindings of opm-simulators](https://github.com/OPM/opm-simulators/blob/master/python/README.md).
 
-For embedding python, see the [documentation](https://opm-project.org/?page_id=1454). 
\ No newline at end of file
+For embedding python, see the [documentation](https://opm-project.org/?page_id=1454).
+
+To enable tooltips for opm_embedded on VSCode (embedded python code): Copy the file "<opm-common-folder>/python/opm_embedded.pyi" in the folder at "python.analysis.stubPath" of VS Code
```

Comment:
I think it should read:
opm_embedded in VSCode ... to the folder defined in variable ...